iot_packet_verifier/src/verifier.rs

```diff
@@ -110,11 +108,14 @@ where
                     })
                     .await
                     .map_err(VerificationError::InvalidPacketWriterError)?;
-                self.config_server
-                    .disable_org(report.oui)
-                    .await
-                    .map_err(VerificationError::ConfigError)?;
             }
+            match remaining_balance {
```

Comment:
could this match and the if clause at line 86 not be merged ?

Reply:
Not cleanly, the match has an if guard